bpf/kmesh/workload/sockops.c

```diff
@@ -106,6 +106,16 @@ static inline void auth_ip_tuple(struct bpf_sock_ops *skops)
     bpf_ringbuf_submit(msg, 0);
 }
 
+// insert an init state to auth_map, indicates that auth is being performed 
+static inline void auth_init(struct bpf_sock_ops *skops)
+{
+    struct bpf_sock_tuple tuple_info = {0};
+    extract_skops_to_tuple(skops, &tuple_info);
+    int err = bpf_map_update_elem(&map_of_auth, &tuple_info, 2, BPF_ANY);
```

Comment:
since we have multi values now, can we define macros for them to prevent hardcoding

Reply:
see macro `AUTH_INIT`, why not use it

Reply:
you are right， will  define macros in public header file